.prettierignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
This project doesn't use Prettier, so no need to add this file.

Reply:
I added this to prevent users with Prettier to accidentally reformat the code, which is very different from whatever format you guys are using. Can you tell which formatter to use? I guess you have it installed globally, which means it's not part of the project infra.

Reply:
I don't use auto-formatters. I don't really want tooling-specific files in the project, is there no other way to prevent this?

Reply:
There's a way, but it's slightly verbose. You have to create a new VS Code workspace directory, apply the `.prettierignore` file there, clone the repository and open the workspace in VS Code. It hurts the developer experience. I for one, use "format on save", so I can't apply changes without disabling the formatter first.